hni-data-model/src/main/java/org/hni/admin/service/dto/NgoBasicDto.java

```diff
@@ -0,0 +1,42 @@
+package org.hni.admin.service.dto;
+
+public class NgoBasicDto {
+	
+	Long user_id;

```

Comment:
Don't use _ in variable names, unless it's really necessary. Please use camel case.